docs/manual/docs/examples/spo/remove-site-scripts.md

```diff
@@ -0,0 +1,64 @@
+# Delete custom SharePoint Site Scripts
```

Comment:
Could you please add the reference to this file in `mkdocs.yml`? Otherwise it won't get included in the generated docs.

Reply:
Added references to mkdoc.yml.... I had to merge dev branch to patch-2 and then do the changes.
I am not sure it was correct way to get latest mkdocs.yml file... . If this is not correct I can create new pull request with only changes to this file. Please advise...

Reply:
If you use docker, you can verify your changes locally, by changing the working directory to `docs/manual` and then running:

```sh
docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:3.1.0
```